auth/api/iam/usersession/data.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2024 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package usersession
+
+import (
+	"errors"
+	"fmt"
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"github.com/nuts-foundation/go-did/did"
+	"github.com/nuts-foundation/go-did/vc"
+	"time"
+)
+
+// Data is a session-bound Verifiable Credential wallet.
+type Data struct {
```

Comment:
extremely generic name. Either name the package `session` and this struct `User` or place everything in the `iam` package and name it `usersession`?

Reply:
moved `usersession` to `http` module, renamed package to `user` and `Data` -> `Session`